api/app/graphql/spree/types/query_type.rb

```diff
@@ -0,0 +1,15 @@
+module Spree
+  module Types
+    class QueryType < ::GraphQL::Schema::Object
+      graphql_name "Query"
+      field :orders, [OrderType], 'Returns all orders', null: false
+
+      def orders
+        Spree::Order.all
```

Comment:
```suggestion
        Spree::Order
```

even for PoC purposes, let's always use scopes, not dump the objects to memory

Reply:
@damianlegawiec but method `all` return relation not array itself or we talk about different things?